components/script/dom/htmliframeelement.rs

```diff
@@ -107,6 +107,11 @@ impl HTMLIFrameElement {
             .unwrap_or_else(|| ServoUrl::parse("about:blank").unwrap())
     }
 
+    fn get_srcdoc(&self) -> String {
```

Comment:
No need for a method, since there's only one caller. We can inline this code in the caller instead.